### ISSUE TITLE
Training new models using 3-channel RGB videos

### ISSUE THREAD

**Issue by Ayush-00** [ShiqiYu/OpenGait#64]
## System information (version)
- Pytorch => 1.6.0
- Operating System / Platform => Ubuntu 18.04
- Cuda => 10.2

## Detailed description
Hi!
I am trying to experiment with GaitSet and GaitPart models with a slight change - I am using RGB inputs instead of silhouettes. I changed the number of in_channels in the config files and also the transform to BaseRgbTransform. 

1. However, when I train on GaitPart, I get a lot of warnings in between about the loss being very low - 
`[WARNING]: Find the loss sum less than 1e-9 but the training process will continue!`

2. While training GaitSet, the loss reported in most iterations is `nan`.

After training, while testing these models, I get 0% recognition accuracy in both cases.
Can you please suggest why this might be happening and what can I do to solve this?

## Issue submission checklist
 - [ ✔️ ] I checked the problem with documentation, FAQ, issues, and have not found solution.

**Comment by aleeyang** [comment]
I also try to multi-channel input instead of silhouette image.I would like to share my result later

**Comment by aleeyang** [comment]
what is your pkl shape？is it T*Channel(rgb)*H*W?

**Comment by ChaoFan996** [comment]
Refer to [GaitEdge](https://github.com/ShiqiYu/OpenGait/tree/master/configs/gaitedge) please.
